pandas/tests/io/test_parquet.py

```diff
@@ -655,6 +655,8 @@ def test_read_empty_array(self, pa, dtype):
                 "value": pd.array([], dtype=dtype),
             }
         )
+        if Version(pyarrow.__version__) == "10.0.1":
+            pytest.skip("skip the pyarrow version '10.0.1'")
```

Comment:
```suggestion
        pytest.importorskip("pyarrow", "10.0.1")
```

And similar below

Reply:
thanks for your help. It seems that it doesn't work when using `pytest.importorskip("pyarrow", "10.0.1")`, but it works properly with `pytest.importorskip("pyarrow", "11.0.0")`